api_app/service_bus/resource_request_sender.py

```diff
@@ -28,8 +28,10 @@ async def send_resource_request_message(resource: Resource, operations_repo: Ope
     # add the operation to the db
     if action == RequestAction.Install:
```

Comment:
should we change this to more of a switch style statement rather than if/else?

Reply:
How so? Swtich statements aren't available in this version of python